### PR TITLE
Generate shopping list

### PR DESCRIPTION
**Pull request by benjamintautfest**
Hi all, added shopping list functionality. Have fun reviewing

### CODE REVIEW

File: meal-planner/src/components/Footer.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+                    />
+                </Link>
+            </div>
+        </FooterStyled>
+    )
+}
+
+const FooterStyled = styled.footer`
+    position: relative;
+    //z-index: 0;
```
Comment (FabianAlleblas) [inline meal-planner/src/components/Footer.js:27]: I would remove comments

File: meal-planner/src/components/Header.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import styled from 'styled-components'
+import logo from '../assets/logo.svg'
```
Comment (FabianAlleblas) [inline meal-planner/src/components/Header.js:2]: If you use a SVG and want to see a SVG tag in your HTML, you could also import it as a ReactComponent...

`import {ReactComponent as Logo} from ...`
`<LogoStyled/>`
`const LogoStyled = styled(Logo)..`

File: meal-planner/src/components/RecipePage.js [hunk 3]
```diff
@@ -26,13 +35,18 @@ export default function RecipePage({
 
 const SectionStyled = styled.section`
     background: #800000fe;
```
Comment (FabianAlleblas) [inline meal-planner/src/components/RecipePage.js:37]: No variables in your globalstyle? Not judging... just observing... 😂

**Review by FabianAlleblas — COMMENTED** [review]
Nice one 🚀 🙌

File: meal-planner/src/components/RecipePage.js (hunk 3 above)
Comment (benjamintautfest) [reply meal-planner/src/components/RecipePage.js:37]: :-)

**Review by dajanaheinze — COMMENTED** [review]
there is nothing more to add!!!

**Pull request merged**